maestro-shell/rpm/maestro-shell.spec

```diff
@@ -31,8 +31,8 @@ An interactive shell for controlling maestro locally on deviceOS.
 %gobuild -o %{gobuilddir}/bin/%{name} %{goipath}
 
 %install
-install -vdm 0755                            %{buildroot}/%{_bindir}/pelion/
-install -vpm 0755 %{gobuilddir}/bin/*        %{buildroot}/%{_bindir}/pelion/
+install -vdm 0755                            %{buildroot}/%{_libdir}/pelion/bin/
```

Comment:
Interesting.  Any idea how I can modify the [maestro-shell script](https://github.com/PelionIoT/distro-pelion-edge/blob/3b40d73f64d0aea2ed8be8440899a4742a9a6e4a/maestro-shell/rpm/files/maestro-shell) to account for that?

Reply:
> Interesting. Any idea how I can modify the [maestro-shell script](https://github.com/PelionIoT/distro-pelion-edge/blob/3b40d73f64d0aea2ed8be8440899a4742a9a6e4a/maestro-shell/rpm/files/maestro-shell) to account for that?

@kbgl To clarify this question, how can I modify the maestro-shell script to find the maestro-shell binary if it could be installed in either lib or lib64?  Is there a macro I should use in the script, like %LIB%/pelion/bin/maestro-shell, or do I need to add an if-statement like  `if [ -e /usr/lib/pelion/bin/maestro-shell ]; then ...; elif [ -e /usr/lib64/pelion/bin/maestro-shell ];  ...`

Reply:
I would use just `/usr/lib/`. It is done this way also for Centos/RHEL system packages (for example systemd binaries - all are in `/usr/lib`). There is no variable for `/usr/lib` (you can get existing variables by running `rpm --showrc` command).

Reply:
ok thanks!